zerver/lib/message.py

```diff
@@ -180,7 +180,7 @@ def save_message_rendered_content(message: Message, content: str) -> str:
 
 class MessageDict:
     @staticmethod
-    def wide_dict(message: Message) -> Dict[str, Any]:
+    def wide_dict(message: Message, realm_id: Optional[int]=None) -> Dict[str, Any]:
```

Comment:
I don't understand -- this seems to ignore the `realm_id` parameter; why is it passed?